jenkinsapi_tests/unittests/test_build_scm_git.py

```diff
@@ -51,9 +51,9 @@ def test_git_revision_branch(build):
     assert build.get_revision_branch()[0]['name'] == 'origin/unstable'
 
 
-def test_git_repo_url(self):
+def test_git_repo_url(build):
     """
     Can we extract git repo url for a given build
     """
-    self.assertIsInstance(self.b.get_repo_url(), six.string_types)
-    self.assertEquals(self.b.get_repo_url(), 'https://github.com/salimfadhley/jenkinsapi.git')
+    build.assertIsInstance(build.get_repo_url(), six.string_types)
```

Comment:
We are not using `nosetests`, thus there is no such method as `assertIsInstance` or `assertEquals`. The test function above this new one can be used as example of `pytest` assertions.

Reply:
ahh fixed and retriggered